src/binance/bnbPlugin.js

```diff
@@ -73,11 +74,33 @@ export class BinancePlugin extends CurrencyPlugin {
 
   async parseUri(
     uri: string,
-    currencyCode?: string,
+    currencyOptions?: string | EdgeParseUriOptions,
     customTokens?: Array<EdgeMetaToken>
   ): Promise<EdgeParsedUri> {
     const networks = { binance: true }
 
+    const currencyCode =
+      typeof currencyOptions === 'string'
+        ? currencyOptions
+        : currencyOptions.currencyCode
+    if (
+      typeof currencyOptions === 'object' &&
+      currencyOptions !== null &&
+      currencyOptions.fromFio
+    ) {
+      let noNetwork = true
+      for (const networkName in networks) {
+        if (uri.indexOf(networkName) === 0) {
+          noNetwork = false
+          break
+        }
+      }
+
+      if (noNetwork) {
```

Comment:
Also need to check that the URI does not begin with any scheme at all. Otherwise it will put the binance scheme on top of the scheme it already has